.gitlab-ci.yml

```diff
@@ -0,0 +1,35 @@
+image: espressomd/buildenv-espresso-ubuntu-cuda:python
+build_and_test:
+  stage: build
+  script:
+    - bash maintainer/cuda_build.sh
+
+
+sphinx_documentation:
+  stage: deploy
+  only:
+    - doc
```

Comment:
Shouldn't this be only: python? We don't want deployments of pull requests or stable branches, but there is no doc branch.